test/Riok.Mapperly.IntegrationTests/Models/TestObject.cs

```diff
@@ -45,6 +45,8 @@ public TestObject(int ctorValue, int unknownValue = 10, int ctorValue2 = 100)
 
         public TestObjectNested? NestedNullableTargetNotNullable { get; set; }
 
+        public TestObjectExtension? Extension { get; set; }
```

Comment:
I'm not sure about the naming here... Extension in C# usually refer to a different concept (AFAIK there are also proposals to allow property extensions which would then be even more confusing here...). Probably use a name like `NestedObject` or similar? (Also applies to the name of the class and other properties in the integration test classes).